tests/reports.test.js

```diff
@@ -0,0 +1,345 @@
+const chai = require('chai');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const chai = require("chai");
```